doc/man3/SSL_CONF_cmd.pod

```diff
@@ -448,7 +448,7 @@ The value is a directory name.
 
 =item B<SSL_CONF_TYPE_NONE>
 
-The value string is not used e.g. a command line option which doesn't take an
+The value string is not used e.g. a command line option which does not take an
```

Comment:
side note: FreeBSD style mandates commas around "e.g." and "i.e.".  But I did not intend for this to become a rathold of applying FreeBSD documentation style to our tree!